Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -323,9 +323,7 @@ static bool canAccessWebInspectorMachPort()
 
     if (m_relayConnection) {
         m_shouldReconnectToRelayOnFailure = true;
-
-        // Send a simple message to make sure the connection is still open.
-        m_relayConnection->sendMessage(@"check", nil);
```

Comment:
do we need any sort of backwards compatibility for this?

NIT: Can we come up with a better name for `WIRCheckMessage`?  Perhaps `WIRPingMessage` (and `WIRPongMessage`) to match naming used in other similar things (e.g. `WebSocket`)?

Reply:
Nothing to be done for backward compatibility here afaict since this existing "check" message was previously just unhandled on the other end. The real fix is in handling the successful ping below.